node_script/wrapper.py

```diff
@@ -17,13 +18,62 @@
 from detic_ros.msg import SegmentationInfo
 
 
+_cv_bridge = CvBridge()
+
+
+@dataclass(frozen=True)
+class InferenceRawResult:
+    segmentation_raw_image: np.ndarray
+    class_indices: List[int]
+    scores: List[float]
+    visualization: Optional[VisImage]
+    header: Header
+    class_names: List[str]
+
+    def get_ros_segmentaion_image(self) -> Image:
+        seg_img = _cv_bridge.cv2_to_imgmsg(self.segmentation_raw_image, encoding="32SC1")
+        seg_img.header = self.header
+        return seg_img
+
+    def get_ros_debug_image(self) -> Image:
+        message = "you didn't configure the wrapper so that it computes the debug images"
+        assert self.visualization is not None, message
+        debug_img = _cv_bridge.cv2_to_imgmsg(
+            self.visualization.get_image(), encoding="rgb8")
+        debug_img.header = self.header
+        return debug_img
+
+    def get_ros_debug_segmentation_img(self) -> Image:
+        human_friendly_scaling = 255 // self.segmentation_raw_image.max()
+        new_data = (self.segmentation_raw_image * human_friendly_scaling).astype(np.uint8)
+        debug_seg_img = _cv_bridge.cv2_to_imgmsg(new_data, encoding="mono8")
+        assert self.header is not None
```

Comment:
Should we add this assertion on other methods as well?
Or does the typing `header: Header` (as opposed to `Optional[Header]`) already ensure that the header is never None?

Reply:
True. header is ensured to be not None, so we don't need this assertion
I will fix